pkgs/by-name/wa/wayfreeze/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  libxkbcommon
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "wayfreeze";
+  version = "0-unstable-2024-05-23";
+
+  src = fetchFromGitHub {
+    owner = "Jappie3";
+    repo = pname;
+    rev = "069dea0b832bd5b7a7872a57bd53f51cd377f206";
+    hash = "sha256-3btFzZbkHT6kBBA3M7OwFsD710VpMiHSXIpHmvCD/es=";
+  };
+
+  cargoHash = "sha256-3OjZhWAgfmMZ0OGeRawk3KZpPqz1QCVkwsyGM+E7o88=";
+
+  cargoLock.lockFile = ./Cargo.lock;
+
```

```suggestion
  cargoHash = "sha256-3OjZhWAgfmMZ0OGeRawk3KZpPqz1QCVkwsyGM+E7o88=";

```

You don't need both `cargoHash` and `cargoLock.lockFile`.
`cargoLock.lockFile` is needed when the upstream `Cargo.lock` lists git dependencies, because those aren't supported by `cargoHash` (yet).

But this isn't the case here, so please remove `cargoLock.lockFile` and your vendored `Cargo.lock`.